deployment.Rmd

```diff
@@ -58,6 +58,22 @@ library(sparklyr)
 sc <- spark_connect(master = "yarn-client")
 ```
 
+or for more customized configuration:
+
+```{r}
+library(sparklyr)
+sc <- spark_connect(master="yarn",
+                    config = list(
```

Comment:
I'd really like to keep configuration within config.yml files rather than inline within the code. This is because config.yml includes support for "profiles" (e.g. default, test, production, etc.) which will allow the same code to work in multiple environments. Could we re-write this with the config settings documented in being within a standalone yaml file?